service/dap/server.go

```diff
@@ -3264,7 +3264,7 @@ func (s *Session) stacktrace(goroutineID int, g *proc.G) (string, error) {
 	}
 
 	var buf bytes.Buffer
-	fmt.Fprintln(&buf, "Stack:")
+	fmt.Fprintln(&buf, "User Stack:")
```

Comment:
That's a bit misleadinig for system goroutines, isn't it?
Maybe prinit "Stack" and append "(excluding runtime frames)" if you exclude them?

Reply:
removed this part of the change for now.